guides/Usage.md

```diff
@@ -116,6 +116,35 @@ project %ItemUpdated{uuid: uuid} = event, _metadata, fn multi ->
 end
 ```
 
+
+### Using the `project_batch` macro
+
+By configuring the base event handler to use batching and using the `project_batch/2` macro, you can project a batch of events in a single step within a transaction. `project_batch/2` receives a list of `{event, metadata}` tuples for all the events in the batch and a similar single-arity function as `project/3` to affect an `Ecto.Multi` structure.
+
+Note that there is currently no built in way to target a single type of event to be projected, and as such a single `project_batch` macro is expected to gracefully handle (or ignore) any events that it may receive
```

Comment:
Future: could we give a way to pass a selector for the subscription?

Reply:
We can still pass it through the subscription_opts